tabs/tab_baseline_matching.py

```
@@ -16,79 +18,79 @@
-    
+
     n_treated = len(treated)
     n_control = len(control)
-    
+
     if n_treated == 0 or n_control == 0:
         return pd.DataFrame(columns=['Variable', 'SMD'])
-    
+
     for col in cat_cols:
         if col not in df.columns: continue
```

Ruff (0.14.10)

Multiple statements on one line (colon)

(E701)